README.md

```diff
@@ -28,7 +28,9 @@ You can install the extension from one of the following links
 Please ensure that you have at least the **minimum** recommended versions
 
 -   Node - 10.15.0 (Check by running `node --version`) - This is the version being enforced on our builds
--   NPM - 6.4.1 (Check by running `npm --version`)
+-   Yarn - Version >= v1.12.1 (Check by running `yarn --version`)
+
+> In case you dont have yarn, please install from: [Yarn](https://www.npmjs.com/package/yarn)
```

Comment:
https://www.npmjs.com/package/yarn suggests installing Yarn with `npm i yarn` but also links to a [Yarn install page](https://yarnpkg.com/en/docs/install) which provides a variety of options. Do we want to suggest `npm i -g yarn` or point directly to the Yarn install page instead?

Reply:
I changed the link to point to `yarn's` own website instead of `npm` listing to avoid this confusion.